open_academy/__manifest__.py

```diff
@@ -10,7 +10,11 @@
     "depends": [
         "base"
         ],
-    "data": [],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/open_academy_course_views.xml",
+        "views/open_academy_course_menus.xml"
```

Comment:
Please change the file name and its reference here. All the module menus will be here, not just the open.academy.course model.

You can review the [guideline](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming) for naming files.

```suggestion
        "views/open_academy_menus.xml"
```